sdk/appconfiguration/app-configuration/src/featureFlag.ts

```diff
@@ -62,10 +66,12 @@ export const FeatureFlagHelper = {
     }
     const jsonFeatureFlagValue: JsonFeatureFlagValue = {
       id: featureFlag.value.id ?? key.replace(featureFlagPrefix, ""),
-      ...featureFlag.value,
```

Comment:
Were `enabled` and `description` the only fields we would miss be removing this line?

I agree it makes sense to remove the line (or move it above the `id` assignment), just want to ensure we aren't forgetting to set a field that was previously being set.

Reply:
> Were enabled and description the only fields we would miss be removing this line?

![image](https://user-images.githubusercontent.com/10452642/123006590-c467d580-d36c-11eb-9e4a-b58793a1dfc5.png)

This is the feature flag value.

> (or move it above the id assignment)

Moving it would add both displayName and display_name, hence removing.